LayoutTests/fast/forms/select/mac-wk2/inactive-appearance.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html>
     <head>
+        <script>window.enablePixelTesting = true;</script>
```

Comment:
Instead of doing this, what if we dropped `js-test.js` here, and used `testRunner.waitUntilDone()` / `testRunner.notifyDone()`?

Basically:

`jsTestIsAsync = true`;

=>

```
if (window.testRunner)
        testRunner.waitUntilDone();
```

and `finishJSTest();`

=>

```
if (window.testRunner)
        testRunner.notifyDone();
```

Reply:
Made that change!